Computer-SpaceGames/Starship-Takeoff/cpp/src/game.cpp

```diff
@@ -0,0 +1,57 @@
+#include <string>
+#include "game.h"
+
+SpaceTakeoffGame::SpaceTakeoffGame(unsigned int gravity, unsigned int weight)
```

Comment:
Both variables could be `const`. Something that's quite common in C++ when passing by value is having the declaration in the header as `void function(type parameter)` whilst the definition is `void function(const type parameter)` (added `const`) which hides the implementation detail that the value won't be changed in the function (if passing by reference or pointer things then that doesn't work).

Reply:
Could add that to all of the definitions in this file.